src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -112,9 +112,7 @@ const ChannelTitle = ({
   }
 
   const getTitle = () => {
-    const isMyPost = type?.includes('MY_SIGNED_POST') || type?.includes('MY_ANON_POST');
-
-    if (isMyPost) {
+    if (isMe) {
       if (message?.length === 0) return 'Your media post 📸 🖼';
       return 'Your post: ';
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `isMe` is used without being defined in the scope. It seems like it's replacing `isMyPost` from the old hunk, but I don't see where `isMe` is defined or its value set. Please ensure that `isMe` is properly defined and set before this usage.
```diff
-    if (isMe) {
+    const isMyPost = type?.includes('MY_SIGNED_POST') || type?.includes('MY_ANON_POST');
+    if (isMyPost) {
       if (message?.length === 0) return 'Your media post 📸 🖼';
       return 'Your post: ';
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->